### DISCUSSION TITLE
服务器在线聊天报错

### DISCUSSION THREAD

**Discussion by Gary2018X** [GaiZhenbiao/ChuanhuChatGPT#388] [Q&A] [answered]
File "/home/projects/Chuanhu/modules/overwrites.py", line 37, in postprocess
    y[-1] = (convert_user(y[-1][0]), convert_mdtext(y[-1][1]))
NameError: name 'convert_user' is not defined
这是为啥

**Comment by GaiZhenbiao** [comment, marked as answer]
也许是什么时候引入的bug。请更新到最新版脚本。我试了试没问题。

**Reply by Gary2018X** [reply]
应该是的 我看最新的代码没有了
